code/game/gamemodes/objective_items.dm

```diff
@@ -51,19 +51,19 @@
 	excludefromjob = list("Captain")
 
 /datum/objective_item/steal/magboots
-	name = "the chief engineer's advanced magnetic boots."
+	name = "the Chief Engineer's advanced magnetic boots."
 	targetitem =  /obj/item/clothing/shoes/magboots/advance
 	difficulty = 5
 	excludefromjob = list("Chief Engineer")
 
 /datum/objective_item/steal/capmedal
-	name = "the medal of captaincy."
+	name = "the Medal of Captaincy."
```

Comment:
Why is this capitalized? The actual item is just called ``"medal of captaincy"``

Reply:
Hmm for some reason i thought it was capitalized in game i can fix that after my work meeting.